integration/spark/shared/src/main/java/io/openlineage/spark/agent/facets/SparkPropertyFacet.java

```diff
@@ -0,0 +1,21 @@
+package io.openlineage.spark.agent.facets;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.openlineage.client.OpenLineage;
+import io.openlineage.spark.agent.Versions;
+import java.util.Map;
+
+public class SparkPropertyFacet extends OpenLineage.DefaultRunFacet {
```

Comment:
Why are you trying to introduce this facet in the PR? 
I thought this should be part of 1717.

Reply:
something bad happened here ...